pkg/k8s/chart/helmrenderer.go

```diff
@@ -24,7 +24,7 @@ type HelmRenderer struct {
 
 func NewHelmRenderer(chartPath string, logger *zap.SugaredLogger) (Renderer, error) {
 	if !file.DirExists(chartPath) {
-		return nil, fmt.Errorf("HELM chart directory '%s' not found", chartPath)
+		return nil, fmt.Errorf("HELM chart directory '%s' not found", chartPath) //nolint: goerr113
```

Comment:
Why could this one not be fixed?

Reply:
Because this does not make sense to fix. I mean this is the only part where we use variable in error message.

Reply:
Can you add the reasoning with this format: `//nolint:<linter> // <reasoning>` ?

Reply:
Sure. :-)